### DISCUSSION TITLE
Create Android app

### DISCUSSION THREAD

**Discussion by 66Ton99** [pear-devs/pear-desktop#1263] [Ideas]
Would be nice to create port for Android phones :)

**Comment by Araxeus** [comment]
You should try out Youtube Vanced (they also have a youtube music app)
https://vancedapp.com/

its really a must have

**Comment by 66Ton99** [comment]
Vanced doesn't have volume normalisation!  :(

**Comment by lg-nineoneninethreenine** [comment]
But, wait, YouTube Music exists as an official app on Android, why do you want to make a port for an app that already exists on a platform?

**Comment by Araxeus** [comment]
> But, wait, YouTube Music exists as an official app on Android, why do you want to make a port for an app that already exists on a platform?

Because of all the additional features? (Play in background / screen closed, Disable Ads, and much more)
You should really try out vanced.

@im-coder-lg don't you enjoy some of the wonderful plugins in this app? 😋

**Comment by senolfndk** [comment]
try this you can download mp3 or video https://ymusic.io/

**Comment by lg-nineoneninethreenine** [comment]
> @im-coder-lg don't you enjoy some of the wonderful plugins in this app?

Yes, I like a notification on my next song, a no sign-in button, and some more plugins. But I don't understand this, why do you need a YouTube Music app for Android, when you already have it? You could set up a nifty software called Pi-Hole to block ads on all apps, but what's the point of recreating an existing app?

Edit: I understood, due to the plugins. But it'd just overload Android, right?

**Comment by Araxeus** [comment]
> Because of all the additional features? (Play in background / screen closed, Disable Ads, Sponsorblock, and much more)

https://youtubevanced.com/#features

**Comment by stefanschlipfi** [comment]
Use this app instead [Vanced APP](https://vancedapp.com/)

**Comment by Araxeus** [comment]
> Use this app instead [Vanced APP](https://vancedapp.com/)

thats literally what I said in my last 3 messages? O_o

**Comment by lg-nineoneninethreenine** [comment]
Yeah, I don't need an app for my phone, but I definitely can use one for the laptop, so Vanced doesn't seem a good idea for phones(for me).

**Comment by Mleekush** [comment]
Well now, since Vanced was discontinued, I would like to have an alternative for the Vanced Music once it eventually gets outdated. I would also love to have something to control my music from phone, like Spotify.

**Comment by briantho** [comment]
> Well now, since Vanced was discontinued, I would like to have an alternative for the Vanced Music once it eventually gets outdated. I would also love to have something to control my music from phone, like Spotify.

I too think that this would be quite useful, not just for android on the phone, but also android tv. android tv is something that vanced never supported, and now that's its discontinued never will support. Right now I've got this software installed on a htpc desktop, but i would absolutely love to get this working on a fire tv. What do the devs say? now that vanced is discontinued, what about actually doing this? or is this simply a pipe dream....

**Comment by MulverineX** [comment]
Everyone, [https://github.com/revanced](https://github.com/revanced)! They will be continuing the YTM mod and it will be open source this time! This project (th-ch/youtube-music), is completely irrelevant as it is web-based and the android app is a java/flutter program. Even if Vanced wasn't being continued by this group linked, a YTM Android mod request would still not belong here.

**Comment by Zo-Bro-23** [comment]
Although not an Android app, #1094 introduces an Android remote that allows you to control songs using your phone.
